src/DataFixtures/ORM/LoadVotingPlatformElectionData.php

```diff
@@ -359,6 +359,7 @@ public function getDependencies()
     {
         return [
             LoadAdherentData::class,
```

Comment:
`LoadAdherentData` is no longer required here, because `LoadCommitteeData` depends on `LoadAdherentData`